packages/gatsby/src/datastore/in-memory/indexing.ts

```diff
@@ -63,109 +62,6 @@ export interface IFilterCache {
 }
 export type FiltersCache = Map<FilterCacheKey, IFilterCache>
 
-/**
- * Get all nodes from redux store.
- */
-export const getNodes = (): Array<IGatsbyNode> => getDataStore().getNodes()
-
-/**
- * Get node by id from store.
- */
-export const getNode = (id: string): IGatsbyNode | undefined =>
-  getDataStore().getNode(id)
-
-/**
- * Get all nodes of type from redux store.
- */
-export const getNodesByType = (type: string): Array<IGatsbyNode> =>
-  getDataStore().getNodesByType(type)
-
-/**
- * Get all type names from redux store.
- */
-export const getTypes = (): Array<string> => getDataStore().getTypes()
-
-/**
- * Determine if node has changed.
- */
-export const hasNodeChanged = (id: string, digest: string): boolean => {
-  const node = getDataStore().getNode(id)
-  if (!node) {
-    return true
-  } else {
-    return node.internal.contentDigest !== digest
-  }
-}
-
-/**
- * Get node and save path dependency.
- */
-export const getNodeAndSavePathDependency = (
-  id: string,
-  path: string
-): IGatsbyNode | undefined => {
-  const node = getNode(id)
-
-  if (!node) {
-    console.error(
-      `getNodeAndSavePathDependency failed for node id: ${id} as it was not found in cache`
-    )
-    return undefined
-  }
-
-  createPageDependency({ path, nodeId: id })
-  return node
-}
-
-type Resolver = (node: IGatsbyNode) => Promise<any> // TODO
-
-export const saveResolvedNodes = async (
-  nodeTypeNames: Array<string>,
-  resolver: Resolver
-): Promise<void> => {
-  for (const typeName of nodeTypeNames) {
-    const nodes = getDataStore().getNodesByType(typeName)
-    if (!nodes || !nodes.length) continue
-
-    const resolvedNodes = new Map()
-    for (const node of nodes) {
-      const resolved = await resolver(node)
-      resolvedNodes.set(node.id, resolved)
-    }
-    store.dispatch({
-      type: `SET_RESOLVED_NODES`,
-      payload: {
-        key: typeName,
-        nodes: resolvedNodes,
-      },
-    })
-  }
-}
```

Comment:
Moved this method to `schema/node-model.js` (where the only usage of this method is)